api.md

```diff
@@ -1669,14 +1669,15 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
          [ optional grouping exprs>]
 [HAVING ...]
 ```
+Note that only hypertable and only one is allowed in `FROM` clause.
```

Comment:
Current sentence reads a bit strange. I suggest:
```suggestion
Note that only a single hypertable is supported in the `FROM` clause.
```

Reply:
I agree it is a little bit strange, but the suggestion might not emphasise `hypertable` strong enough. But I will go for it.